packages/babel-plugin-proposal-private-property-in-object/test/fixtures/to-native-fields/accessor/output.js

```diff
@@ -0,0 +1,14 @@
+var _FooBrandCheck = new WeakSet();
+
+class Foo {
+  constructor() {
+    _FooBrandCheck.add(this);
+  }
+
+  get #foo() {}
+
+  test(other) {
+    return _FooBrandCheck.has(other);
```

Comment:
Q: Is `_FooBrandCheck.has(other)` equivalent to `other instanceof Foo`?

Reply:
No, it's different when other is a subclass of `Foo`

Reply:
If `Bar`, a subclass of `Foo`, has `super()` in its constructor, does it imply `#foo in bar` is `true` where `bar` is an instance of `Bar`?

Reply:
Oh I was confusing this test with the private fields one. `instanceof` is different because it checks the prototype:

```js
class Base {
  constructor() {
    return {};
  }
}

class Foo extends Base {
  #foo;

  static check(obj) {
    return #foo in obj;
  }
}

new Foo() instanceof Foo; // false
Foo.check(new Foo); // true
```

```js
class Foo {
  #foo;

  static check(obj) {
    return #foo in obj;
  }
}

let foo = new Foo();
foo.__proto__ = {};
foo instanceof Foo; // false
Foo.check(foo); // true
```